packages/mobile/tsconfig.json

```diff
@@ -11,7 +11,8 @@
             "@contexts/*": ["../shared/lib/contexts/*"],
             "@core/*": ["../shared/lib/core/*"],
             "@features/*": ["../shared/features/*"],
-            "@lib/*": ["../shared/lib/*"]
+            "@lib/*": ["../shared/lib/*"],
+            "@mobile/*": ["./*"]
```

Comment:
I added an issue to make this work, I would remove it from here and try making it work in a different PR